<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::InputKeyboard"/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;OCR A&quot;;
color: rgb(0, 255, 0);
background-color: rgb(40, 40, 40);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QListWidget" name="WeeklyToDo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ImportantDates">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="DailyEvents">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="FinancialToDo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImportantDatesLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;
text-align:center;
background-color:regb(255,0,0);
color:white;</string>
    </property>
    <property name="text">
     <string>Important Dates</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WeeklyToDoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;
text-align:center;
background-color:regb(255,0,0);
color:white;</string>
    </property>
    <property name="text">
     <string>Weekly ToDo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="DailyAgendaLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;
text-align:center;
background-color:regb(255,0,0);
color:white;</string>
    </property>
    <property name="text">
     <string>Daily Agenda</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FinancialLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;
text-align:center;
background-color:regb(255,0,0);
color:white;</string>
    </property>
    <property name="text">
     <string>Financial ToDo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Sunday">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SUNDAY</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Monday">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MONDAY</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Tuesday">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TUESDAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Wednesday">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>WEDNESDAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Thursday">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>THURSDAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Friday">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FRIDAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Saturday">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SATURDAY</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ImportantEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Enter your new item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FinancialEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Enter your new item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DailyEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Enter your new item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WeeklyEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;OCR A&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Enter your new item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddImportant">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListAdd"/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFinancial">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListAdd"/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddDaily">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListAdd"/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddWeekly">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListAdd"/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
